source/precalculus/source/04-PR/01.ptx

```diff
@@ -542,7 +555,14 @@
               <li> <p> downwards   </p> </li>
               <li> <p> <m>(3,20)</m>; maximum   </p> </li>
               <li> <p> <m>x=3</m>   </p> </li>
-              <li> <p> (placeholder for graph)   </p> </li>
+              <li> <p> <figure width="100%" xml:id="graph-of--5(x-3)^2+20">
```

Comment:
```suggestion
              <li> <p> <figure width="100%" xml:id="graph-of--5(x-3)2+20">
```

Reply:
Should I use a different name? I initially had like "graph-of-part-a" but thought that was too vague

Reply:
I don't mind this kind of name since it is descriptive, but there are strict rules about what characters are allowed in the xml:id.  The parenthesis and + may not be allowed either...